src/proto/generation.proto

```diff
@@ -45,6 +48,7 @@ message Artifact {
     string text = 6;              // Text data, e.g. text prompt
     Tokens tokens = 7;            // Tokenized text data, e.g. GPT tokens
     ClassifierParameters classifier = 11;
+    tensors.Tensor tensor = 14;   // torch.Tensor; RGB 4D tensor, or VAE latents
```

Comment:
"RGB 4D tensor" 4D because expect batch dimension?
Looks cool that is general support for tensors with different shape and dtypes. 😎

Reply:
`api-interfaces` actually submodules this, and builds typescript/javascript support. I didn't want to deal with the toolchain issues in this particular repository.

Reply:
RE: "RGB 4D tensor" -- yes, and to distinguish from VAE latents which are 3 dimensions.